### PR TITLE
hosseini_PlayerFragmentView

### PR DESCRIPTION
**Pull request by hosseiniSeyRo**
design PlayerFragment view

### CODE REVIEW

File: app/src/main/res/layout/custom_exo_playback_control.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        android:layout_marginTop="28dp"
+        android:gravity="center"
+        android:orientation="horizontal"
+        android:textSize="13sp">
+
+        <ImageButton
+            android:id="@id/exo_prev"
+            style="@style/ExoMediaButton.Previous" />
+
+        <!--<ImageButton-->
```
Comment (helloseyedjafari) [inline app/src/main/res/layout/custom_exo_playback_control.xml]: comment nabashe behtare

File: app/src/main/res/layout/custom_exo_playback_control.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                android:id="@id/exo_pause"
+                style="@style/ExoMediaButton.Pause" />
+
+            <ImageButton
+                android:id="@id/exo_play"
+                style="@style/ExoMediaButton.Play" />
+
+        </RelativeLayout>
+
+        <!--<ImageButton-->
```
Comment (helloseyedjafari) [inline app/src/main/res/layout/custom_exo_playback_control.xml]: hamchenin

**Review by helloseyedjafari — DISMISSED** [review]
khobe

File: app/src/main/res/layout/custom_exo_playback_control.xml (hunk 2 above)
Comment (hosseiniSeyRo) [reply app/src/main/res/layout/custom_exo_playback_control.xml]: done

File: app/src/main/res/layout/custom_exo_playback_control.xml (hunk 1 above)
Comment (hosseiniSeyRo) [reply app/src/main/res/layout/custom_exo_playback_control.xml]: done

**Review by helloseyedjafari — APPROVED** [review]

**Pull request merged**